main.js

```diff
@@ -1,18 +1,17 @@
-const { ipcMain: ipc, app, nativeTheme } = require('electron');
+const { ipcMain: ipc, app, nativeTheme } = require('electron/main');
 const { menubar } = require('menubar');
 const { autoUpdater } = require('electron-updater');
 const { onFirstRunMaybe } = require('./first-run');
 const path = require('node:path');
 
-require('@electron/remote/main').initialize();
```

Comment:
@bmulholland - thoughts on if this is OK to remove now.  I know you were cleaning this up via https://github.com/gitify-app/gitify/pull/654

Reply:
If it doesn't work it doesn't, but otherwise I see no reason to not remove it!

Reply:
I don't want to pull the trigger on this yet.  left a TODO referencing #650 for future us